<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@838.va_3a_087b_4055b">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@2759.v87459c4eea_ca_">
    <parentIds>
      <string>6</string>
    </parentIds>
    <id>7</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@2759.v87459c4eea_ca_">
      <arguments>
        <entry>
          <string>script</string>
          <string>eho &quot;Pipeline is executing&quot;</string>
        </entry>
      </arguments>
      <sensitiveVariables/>
      <isUnmodifiedBySanitization>true</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@1192.v2d0deb_19d212">
      <startTime>1663759486503</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <wf.a.ErrorAction plugin="workflow-api@1192.v2d0deb_19d212">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 127</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:664)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:610)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:554)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)</trace>
          <trace>java.base/java.lang.Thread.run(Unknown Source)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>